pkg/workspace_repo/api_client.go

```diff
@@ -25,17 +25,19 @@ const (
 )
 
 type WorkspaceRepo struct {
-	url      string
-	org      string
-	hostAddr string
-	tfc      *tfe.Client
-	dryRun   util.DryRunType
+	url         string
+	org         string
+	hostAddr    string
+	tfc         *tfe.Client
+	dryRun      util.DryRunType
+	interactive bool
```

Comment:
why is interactive a property of the workspace?

Reply:
I think we had some ctx code that would set properties like `isCI()` and this feels similar